conjure/controllers/deploystatus/common.py

```diff
@@ -21,7 +21,12 @@ def wait_for_applications(script, msg_cb):
             count = 0
             while rerun:
                 sh = run_script(script)
-                result = json.loads(sh.stdout.decode('utf8'))
+                try:
+                    result = json.loads(sh.stdout.decode('utf8'))
+                except json.decoder.JSONDecodeError as e:
+                    app.log.exception(sh.stdout.decode())
+                    raise Exception(sh)
```

Comment:
sh is the result of run_script which is just a subprocess run (sh meaning shell i guess), that exception will print the result object of subprocess run